i18n/fr/docusaurus-plugin-content-docs/current/prompt_hacking/intro.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 0
+---
+
+
+
+# 🟢 Introduction
+
+import Lock from '@site/static/img/lock.webp';
+
+<div style={{textAlign: 'center'}}>
+  <img src={Lock} style={{width:"30%"}}/>
+
+</div>
+
+
+Le prompt hacking (piratage de prompt ou le hacking de prompt) est un terme utilisé pour décrire un type d'attaque qui exploite les vulnérabilités des %%LLMs|LLM%%, en manipulant leurs entrées ou prompts. Contrairement au hacking traditionnel, qui exploite généralement les vulnérabilités logicielles, le hacking de prompt repose sur la création soignée de prompts pour tromper le LLM et le faire réaliser des actions non intentionnelles.
```

Comment:
i reversed my choice of wording from "piratage" to "hacking", but left "piratage" here just for searchability/discoverability